.github/workflows/benchmark.yml

```diff
@@ -49,12 +49,11 @@ jobs:
             # Set up ccache configs .
             mkdir -p .ccache
             ccache -sz -M 5Gi
-            
+
             # We can not use setup-python as we are running in a container
-            # TODO add to docker image and remove here
+            # TODO remove after #3904 is merged
```

Comment:
nit: lets remove this comment, lets use gh issues etc for tracking work.

Reply:
```suggestion
```